backend/Origam.Common/OrigamSettings.cs

```diff
@@ -267,6 +269,9 @@ public override string ToString()
 
         [Category("Model Connection")]
         public bool CheckFileHashesAfterModelLoad { get; set; } = true;
+        
+        [Category("SmsService")] 
+        public string SmsService { get; set; } = "Origam.Sms.DebugSmsService, Origam.Sms";
```

Comment:
Origam.Sms library doesn't contain DebugSmsService, as it was discussed previously, we should implement somewhere FileSystemSmsService and use it as a default value here.

Reply:
True, i just renamed to FileSystemSmsService and this class was also added to  Origam.Sms project with testing method.

Reply:
FileSystemSmsService will be separated repo and project.